src/components/_auth/SignInSteps/TermsStep.tsx

```diff
@@ -54,7 +55,7 @@ export const TermsStep: React.FC = () => {
         </TermsOverlay>
       </TermsBox>
       <StepFooter>
-        <ContinueButton to={absoluteRoutes.studio.newMembership()} disabled={!hasScrolledToBottom}>
+        <ContinueButton to={`${pathname}?step=membership`} disabled={!hasScrolledToBottom}>
```

Comment:
Also, could you check if `to={{ search: { [QUERY_PARAMS.SIGN_IN]: 1 } }}` would work instead of providing the pathname?

Reply:
> Also, could you check if `to={{ search: { [QUERY_PARAMS.SIGN_IN]: 1 } }}` would work instead of providing the pathname?

Unfortunately no, `search` needs to be a string but I've created a small utility function that allows doing something similar.